app/views/relationships/create.js.erb

```diff
@@ -0,0 +1,2 @@
+$("#follow_form").html("<%= escape_javascript(render('users/unfollow')) %>");
```

Comment:
change `$("#follow_form").html("<%= escape_javascript(render('users/unfollow')) %>");`
to `$("#follow_form").html("<%= j render 'users/unfollow' %>");`